src/transformStateWithClones.js

```diff
@@ -8,6 +8,49 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const k = [];
+  let stateClone = {
+    ...state,
+  };
+
+  for (const action in actions) {
+    const { type } = actions[action];
```

Comment:
```suggestion
    const { type, extraData, keysToRemove } = action;
```
You can desctucture all the needed props at one time